app/components/transfer/AnnotatedLoader.js

```diff
@@ -25,12 +25,12 @@ export default class AnnotatedLoader extends Component<Props> {
         <div>
           <div className={styles.body}>
 
+            <LoadingSpinner />
```

Comment:
@ladomyriak 
Now this PR does not use `AnnotatedLoader.js` and `AnnotatedLoader.scss` in any way, then is it improvement of it?

Reply:
@sarveshwar-singh  Instead of `AnnotatedLoader.js` we use `.walletLoader` which has gif background image.

Reply:
it was implemented according to your instruction.

Reply:
@ladomyriak 
My point is then why `AnnotatedLoader.js` and `AnnotatedLoader.scss` is changed at all?

Reply:
@sarveshwar-singh  yes, it is improvement according to new designs. Loading dialog has some updates. So I applied it for both cases, for a new case and old AnnotatedLoader.